src/views/Home.vue

```diff
@@ -231,6 +239,16 @@ export default {
     flush(projects) {
       this.$nextTick(() => localStorage.setItem('projects', JSON.stringify(projects)))
       system.saveHostEntries(projects)
+    },
+
+    minimizeWindow() {
+      const { remote } = require('electron')
```

Comment:
Could you move these to the top of the `<script>` section along with the import of `WindowControls` and convert them to ES6 style imports rather than CommonJS?


```
// ES6 Style
import { remote } from 'electron'

// CommonJS Style
const { remote } = require('electron')
```

Typically, in Node projects, you'll see CommonJS used while in most modern front-end stacks (which are typically run through Webpack/Babel, you'll see ES6 modules.